_rules/SC1-2-Video-description-track.md

```diff
@@ -38,51 +38,51 @@ This means that the rule can only provide a pass for these succescriteria if acc
 
 ## Background
 
-- http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc.html
-- http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc-only.html
-- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G78
-- https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/H96
+- [Understanding Success Criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)](http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc.html)
```

Comment:
(cannot comment on the Accessibility support section, but that is what is addressed in this comment)
Please review and correct several misspelling:
discription -> description
succescriteria -> success criteria
accessistive -> assistive